docs/SRS-Volere/SRS.tex

```diff
@@ -301,17 +302,20 @@ \subsection{Glossary of All Terms, Including Acronyms, Used by Stakeholders
   \item CSE: Confined Space Entry
   \item Dry Well: A room which houses industrial equipment such as pumps
     and valves.
+  \item GPS: Global Positioning System
   \item Hazard Assessment Form: A form outlining potential
     hazards at a location.
   \item Hot Works: Work which produces ignition sources.
     Requires an accompanying hot works permit.
   \item HVAC: Heating, Ventilation, and Air Conditiong.
   \item Infor EAM: An Enterprise Asset Management system.
+  \item MySDS: An external application which provides safety data sheets.
   \item PMATS: Plant Maintenance and Technical Services.
   \item PO: Purchase Order.
   \item PPE: Personal Protective Equipment.
   \item SCADA: Supervisory Control and Data Acquisition.
   \item SDS: Safety Data Sheets.
+  \item Sharepoint: A Microsoft platform for collaboration and file storage.
```

Comment:
do we wanna also remove all sharepoint references in this PR too? since we're not doing that stuff anymore